js/share_test.go

```diff
@@ -0,0 +1,173 @@
+/*
```

Comment:
This file should be in `js/modules/k6/data`.

Reply:
I tried this early when I moved it to a package, but I can't really run this whole test and I mostly care about the part that it is accessible by other VUs when you just run a test ... which is really hard to do in a js/modules test

Reply:
Hhmm right, you need `getSimpleRunner`, etc. OK, it's not a big deal.